### PR TITLE
Add Invite Command

### PR DESCRIPTION
**Pull request by JessWalters**
[//]: # (Please try to use this template, feel free to remove sections or add more you may think are relevant.)

## Problem
- To achieve feature parity we need this command

## Solution
- Added the new command

## New Commands added
- Invite Command

## Old Commands affected
- None

#### Does this use the database?
no
#### Does this store user data?
no
#### Does this talk to a 3rd party API or Service?
no

## Reviewers
- [ ] @JessWalters

### CODE REVIEW

**Comment by mrb25** [comment]
Not shady enough

**Comment by demisardonic** [comment]
Please link your local testing results.

File: src/main/java/com/bot/ShardingManager.java [hunk 1]
```diff
@@ -50,6 +51,9 @@ public ShardingManager(int numShards, boolean supportScript) throws Exception {
                     // Battle Royale
                     new BattleRoyaleCommand());
 
+                    // General Commands
+                    new InviteCommand();
```
Comment (demisardonic) [inline src/main/java/com/bot/ShardingManager.java]: nitpick: alphabetize with other commands

File: src/main/java/com/bot/commands/voice/ListMyPlaylistCommand.java [hunk 2]
```diff
@@ -1,6 +1,5 @@
 package com.bot.commands.voice;
 
-import com.bot.Bot;
```
Comment (demisardonic) [inline src/main/java/com/bot/commands/voice/ListMyPlaylistCommand.java]: Why was this removed?

Comment (JessWalters) [reply src/main/java/com/bot/commands/voice/ListMyPlaylistCommand.java]: It was removed because it was unused

File: src/main/java/com/bot/commands/general/InviteCommand.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    }
+
+    @Override
+    protected void execute(CommandEvent commandEvent) {
+        User user = commandEvent.getAuthor();
+        PrivateChannel privateChannel = user.openPrivateChannel().complete();
+        // TODO: This is currently the Test Bot invite link. Before public release change this too the actual bot
+        privateChannel.sendMessage("https://discordapp.com/api/oauth2/authorize?client_id=489251749387960321&permissions=0&redirect_uri=https%3A%2F%2Fwww.official.men&response_type=code&scope=guilds%20identify%20guilds.join%20gdm.join%20rpc%20rpc.api%20messages.read%20bot%20rpc.notifications.read").queue();
+    }
+}
```
Comment (quadespellman) [inline src/main/java/com/bot/commands/general/InviteCommand.java]: Unfinished? Needs work

File: src/main/java/com/bot/ShardingManager.java (hunk 1 above)
Comment (quadespellman) [reply src/main/java/com/bot/ShardingManager.java]: Also remove the comments, code should be self documenting.

**Comment by JessWalters** [comment]
You all a bunch of goofs
